config.py

```diff
@@ -0,0 +1,27 @@
+"""
+Functions to config and initialise the system. Expects environment variables for most
+"""
+
+import json
+import os
+
+
+class Config:
+    ENV = 'prod'
+
+    VERIFY_DATA_PIPELINE_CONFIG_PATH = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), '..', 'verify-data-pipeline-config'))
+    PIWIK_PERIOD = 'week'
+    PIWIK_LIMIT = '-1'
+    PIWIK_BASE_URL = 'https://analytics-hub-prod-a-dmz.ida.digital.cabinet-office.gov.uk/index.php'
```

Comment:
Potentially this should vary depending on `env` also, so maybe we need a TODO here as well :)